index.js

```diff
@@ -31,6 +31,10 @@ if (process.env.hasOwnProperty('GCLOUD_DEBUG_LOGLEVEL')) {
 if (process.env.hasOwnProperty('GCLOUD_DEBUG_DISABLE')) {
   config.enabled = false;
 }
+if (process.env.hasOwnProperty('GCLOUD_DEBUG_RELATIVE_REPOSITORY_PATH')) {
```

Comment:
I forget that debug doesn't currently have a mechanism to accept config on startup. We'll have to keep the env. var. until we can fix that.